data/human/news.txt

```diff
@@ -2140,10 +2161,11 @@ news "label on ship"
 			"Angles morts - Blind spots from cockpit."
 			"Stay clear of ramscoop intake."
 			"Stay out of thruster exhaust stream."
-			"Stay away form thruster outlets during takeoff."
+			"Stay away from thruster outlets during takeoff."
 			"Door opens outwards."
 			"Remove red ribbons before flight."
-			"Nuclear reactor compartiment, NEA certification required for servicing this part of the ship."
+			"Nuclear reactor compartment, NEA certification is required for servicing this part of the ship."
+			"Compartiment du reacteur nucleaire, la certification du NEA est requise avant l'entretien."
```

Comment:
I like having it in French, but realistically ~~(don't ban me)~~ I kind of doubt that Francophone languages would be all that common by the 31st century. Could work, but you'd probably want to add a couple more languages too.

Also, I *think* it should be done like this, possibly with `l'` instead of `la` (since I'd expect it to be agence de l'énergie nuclearie).
```suggestion
			"Compartiment du reacteur nucleaire, la certification de la AEN est requise avant l'entretien."
```